packages/storefront/src/components/Navigation.vue

```diff
@@ -54,8 +57,8 @@
       return this.$store.getters.isSearchActive;
     }
 
-    public getRoute(category: string, page: string): string {
-      return `/${paramCase(category)}/${paramCase(page)}`;
+    public getRoute(category: string, page: string, tab?: string): string {
+      return `/${paramCase(category)}/${paramCase(page)}${tab ? '/' + paramCase(tab) : ''}`;
```

Comment:
Might be easier to collect the 3 parameters in an array, filter out the empty/invalid one and join it with `/`.